tests/generated-methods/build.gradle.kts

```
@@ -0,0 +1,37 @@
+import org.jetbrains.kotlin.gradle.dsl.KotlinCompile
+import org.jetbrains.kotlin.gradle.dsl.KotlinVersion
+
+plugins {
+  id("org.jetbrains.kotlin.jvm")
+  id("apollo.test")
+  id("com.apollographql.apollo3")
+}
+
+dependencies {
+  implementation(libs.apollo.runtime)
+  testImplementation(libs.kotlin.test)
+  testImplementation(libs.junit)
+}
+
+apollo {
+  service("kotlin19") {
+    packageName.set("generatedMethods.kotlin19")
+    generateDataBuilders.set(true)
+    generateMethods.set(listOf("toString", "equalsHashCode", "copy"))
+  }
+}
+
+//kotlin {
+//  compilerOptions {
+//    languageVersion.set(KotlinVersion.KOTLIN_1_9)
+//    apiVersion.set(KotlinVersion.KOTLIN_1_9)
+//  }
+//}
+
+
+tasks.withType(KotlinCompile::class.java).configureEach {
+  kotlinOptions {
+    apiVersion = "1.9"
+    languageVersion = "1.9"
+  }
+}
```

We run tests using Kotlin 1.6 because this is what we claim compatibility with. `enum` is the only exception because it has code to support 1.9 `entries`. Since this test is not about `entries`, I'd rather have it use the minimum Kotlin version:

```suggestion
apollo {
  service("service") {
    packageName.set("generatedMethods")
    languageVersion.set("1.5")
    generateMethods.set(listOf("toString", "equalsHashCode", "copy"))
  }
}
```